File: notify/listServices.js
// -----------------------------------------------------------------------------
console.log("+++ List Services.");

const accountSid = process.env.MASTER_ACCOUNT_SID;
const authToken = process.env.MASTER_AUTH_TOKEN;
const client = require('twilio')(accountSid, authToken);

client.notify.services.each(services =>
    console.log("+ " + services.sid
            + " " + services.friendlyName
            )
);

// -----------------------------------------------------------------------------

File: notify/listServices.py
print "+++ List Services."
import os
from twilio.rest import Client

# Documentation: https://www.twilio.com/docs/notify/api/services
# ------------------------------------------------------------------------------

account_sid = os.environ.get("MASTER_ACCOUNT_SID")
auth_token = os.environ.get("MASTER_AUTH_TOKEN")
client = Client(account_sid, auth_token)

services = client.notify.services.list()
for service in services:
    print("+ " + service.sid + " " + service.friendly_name)

# ------------------------------------------------------------------------------
print("+ End of list.")
